### PR TITLE
Finished base for TriggerService (not tested)

### PR DESCRIPTION
**Pull request by MitchPetersen**
Implemented and changed methods needed to conclude trigger service implementation. Needs review and testing.

### CODE REVIEW

File: CacheService/docker_context/apiListeners.py [hunk 1]
```diff
@@ -38,8 +38,8 @@ async def initAuditHandler(app, loop):
 async def initLegacyStockHandler(app, loop):
     logger.debug('Creating legacy stock handler')
     app.config['legacyStock'] = LegacyStockServerHandler(
-        config.LEGACY_STOCK_SERVER_IP,
-        config.LEGACY_STOCK_SERVER_PORT,
+        config.DUMMY_STOCK_SERVER_IP,
```
Comment (MitchellGRead) [inline CacheService/docker_context/apiListeners.py]: not sure if this was supposed to be left this way

File: TriggerService/docker_context/TriggerHandler.py [hunk 2]
```diff
@@ -44,39 +46,92 @@ def toTrigger(self, trigger_data):
...
+        if status != 200 or results is None:
+            logger.error(f'Failed during set buy trigger call to DBM: {results}')
+            return "setBuyTrigger failed on DBM call", 500
+
+        trigger_data = results.json
+        trigger_data['command'] = command
+        trigger_data['trigger_price'] = trigger_data.pop('price')
+        trigger = self.toTrigger(trigger_data)
+
+        self.TriggerExecutionManager.addTrigger(trigger)
```
Comment (MitchellGRead) [inline TriggerService/docker_context/TriggerHandler.py]: do we want to have this logic handled in the service logic? This way we don't end up coupling the handler and execution manager. We could return the trigger object from the back, service logic then adds that to the execution and if all is successful returns the result for the api

File: TriggerService/docker_context/TriggerHandler.py [hunk 3]
```diff
@@ -44,39 +46,92 @@ def toTrigger(self, trigger_data):
...
     async def cancelBuyTrigger(self, trans_num, command, user_id, stock_symbol):
-        # TODO make POST request to cache service endpoint
-        # TODO return trigger object for removal in execution manager
-        pass
+        endpoint = '/triggers/buy/cancel'
+        data = {'transaction_num': trans_num, 'user_id': user_id, 'stock_symbol': stock_symbol}
+        trigger = self.TriggerExecutionManager.getTrigger(user_id, stock_symbol, self.convertCommand('SET_BUY_TRIGGER'))
+
+        if trigger:
+            self.TriggerExecutionManager.removeTrigger(trigger)
```
Comment (MitchellGRead) [inline TriggerService/docker_context/TriggerHandler.py]: similar reasons as above

**Review by MitchellGRead — COMMENTED** [review]
Looks good, just couple of comments

File: TriggerService/docker_context/TriggerHandler.py (hunk 2 above)
Comment (MitchPetersen) [reply TriggerService/docker_context/TriggerHandler.py]: Changed to do this.

**Review by MitchellGRead — APPROVED** [review]
Nice work 👍

**Review by MitchellGRead — APPROVED** [review]
looks good

**Pull request merged**